eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -46,7 +46,9 @@ function GetDocsTocDisplayName($pkg) {
   return $displayName
 }
 
-
+if (-not (Test-Path -Path "variable:yqPresent")) {
+  $script:yqPresent = $null  # Initial unset state
```

Comment:
I think we could use `$global` here

Reply:
We could. $script is $global but scoped only to this file. I will check it and swap.

Reply:
I'm going to keep `$script` as there's no real reason for a global variable if the module-level works just fine.